queries/liquidsoap/highlights.scm

```diff
@@ -0,0 +1,84 @@
+(op) @operator
+["and" "or" "not"] @keyword.operator
+
+[
+  "def"
+  "fun"
+  "let"
```

Comment:
is let just used to assign variables? or does it actually do something function-related

If the former it should just be `@keyword`

Reply:
Done